proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -11,11 +11,16 @@ import "google/protobuf/any.proto";
 option go_package                      = "github.com/cosmos/cosmos-sdk/x/authz";
 option (gogoproto.goproto_getters_all) = false;
 
+option (cosmos_proto.declare_interface) = {
+  name: "Authorization",
```

Comment:
This interface has name `Authorization` in package `cosmos.authz.v1beta1`.

This way, all other `{accepts,implements}_interface` that reference `cosmos.authz.v1beta1.Authorization` know that they should come here to see what that interface means.